README.md

```diff
@@ -44,4 +44,48 @@ Basic product configuration, as referenced in the get started guides, can be fou
     implements bidirectional data synchronization between IDM and LDAP.
   * `/{version}/default/ig/basic-sample` - Contains configuration to deploy the simplest 
     possible IG server.
+    
+## Contents
+
+### 6.5 configurations   
+
+Directory                   | Contents
+----------------------------|-------------------------------------------
+----------------------------| **DEFAULT CONFIGURATIONS**
+\6.5\\**default**           | Default out of the box configurations to get started using our products.
+\6.5\default\\**am**        | Empty config which triggers default installation of AM.
+\6.5\default\\**idm**       | 
+\6.5\default\\**ig**        | 
+----------------------------| **CLOUD DEPLOYMENT MODEL**
+\6.5\\**CDM**               | Cloud Deployment Model (Common use ForgeRock Identity Platform deployment).
```

Comment:
Our convention is to have lower case paths, so s/CDM/cdm/

I notice this does not match the paths currently in the repo, so that something needs to be fixed - either the README or the path.